crossplane.yaml

```diff
@@ -3,6 +3,52 @@ kind: Configuration
+
+    friendly-name.meta.crossplane.io: AWS Reference Platform
+    friendly-group-name.meta.crossplane.io/aws.platformref.crossplane.io: AWS
+
+    friendly-kind-name.meta.crossplane.io/compositenetwork.aws.platformref.crossplane.io: Network
+    friendly-kind-name.meta.crossplane.io/network.aws.platformref.crossplane.io: Network
```

Comment:
I wonder if this pattern of giving the XR and its claim the same friendly name makes sense. 🤔